pkg/defaults/defaults.go

```diff
@@ -827,19 +828,20 @@ func ensureImageStreamTag(ctx context.Context, client ctrlruntimeclient.Client,
 	istImport.WithImageStreamLabels()
 
 	// Conflicts are expected
-	if err := client.Create(ctx, istImport); err != nil && !kapierrors.IsConflict(err) {
-		logrus.WithError(err).Warnf("Failed to create imagestreamtagimport for root %s", istImport.Name)
+	if err := client.Create(ctx, istImport); err != nil && !kapierrors.IsAlreadyExists(err) {
+		logrus.WithError(err).Warnf("Failed to create imagestreamtagimport for root %s", isTagRef.ISTagName())
 	}
 
+	name := types.NamespacedName{Namespace: isTagRef.Namespace, Name: isTagRef.Name + ":" + isTagRef.Tag}
 	if err := wait.PollImmediate(5*second, 30*second, func() (bool, error) {
-		if err := client.Get(ctx, ctrlruntimeclient.ObjectKeyFromObject(istImport), &imagev1.ImageStreamTag{}); err != nil {
+		if err := client.Get(ctx, name, &imagev1.ImageStreamTag{}); err != nil {
```

Comment:
The name we used here to poll the imagestreamtag was the one of the testimagestreamtagimport, which is pretty much plain wrong, ref: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/18975/rehearse-18975-pull-ci-openshift-ci-tools-master-unit/1400151288242507776#1:build-log.txt%3A5